src/objects/Points.js

```diff
@@ -136,6 +138,46 @@ Points.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 	}() ),
 
+	updateMorphTargets: function () {
+
+		var geometry = this.geometry;
+		var m, ml, name;
+
+		if ( ! geometry.isBufferGeometry && geometry.morphTargets !== undefined && geometry.morphTargets.length > 0 ) {
```

Comment:
IMO, zhis `if` statement can be deleted since I have only added it in `Mesh` for backwards compatibility reasons.

Reply:
Oh, sorry I missed this comment. Sure, I'll remove it 👍